arctic/tickstore/tickstore.py

```diff
@@ -749,3 +749,16 @@ def max_date(self, symbol):
         res = self._collection.find_one({SYMBOL: symbol}, projection={ID: 0, END: 1},
                                         sort=[(START, pymongo.DESCENDING)])
         return utc_dt_to_local_dt(res[END])
+
+    def min_date(self, symbol):
+        """
+        Return the minimum datetime stored for a particular symbol
+
+        Parameters
+        ----------
+        symbol : `str`
+            symbol name for the item
+        """
+        res = self._collection.find_one({SYMBOL: symbol}, projection={ID: 0, START: 1},
+                                        sort=[(START, pymongo.ASCENDING)])
```

Comment:
do we care about handling any of the below, or we prefer to let the exception bubble up?
 - res is None
 - START in res

maybe:
```
if not res:
     raise NoDataFoundException("No Data found for {}".format(symbol))
```

Reply:
I put this in